### ISSUE TITLE
the top left joystick link is hardcoded

### ISSUE THREAD

**Issue by meleu** [botolo78/RetroPie-Manager#12]
The link at the top left joystick icon is addressed to "10.0.0.112:8000".

**Comment by botolo78** [comment]
Are you sure?
The code from projects/templates/skeleton.html is this

                <li class="name">
                    <p class="title-1">
                        <a href="//{{ SITE.domain }}" title="{{ SITE.name }}" class="logo">
                            <i class="icon-retropie"></i>
                        </a>
                    </p>
                </li>


I didn't try in the last few days but my old version is ok and use my local ip address

**Comment by meleu** [comment]
I've installed from the @HerbFargus RetroPie module ( #4 ). Maybe it's something with it...

**Comment by meleu** [comment]
@botolo78 it happens only with the joystick icon. All the others link are OK. I'll check those templates...

**Comment by meleu** [comment]
I think the problem is with this `SITE.domain`. It allways get the port 8000, and I implemented a way to let the user start RetroPie-Manager listening to another port. The problem occurred when I started it listening to the port 8080.

**Comment by meleu** [comment]
Although I think that changing the listening port is useful, I deleted the `--port` option from `rpmanager.sh` to avoid this problem.

**Comment by botolo78** [comment]
I think it's better to leave the port in the config file. Anyhow I don't see how it could hardcode the ip address since it's never used in the code.

Unless you changed the 0.0.0.0:8000 in the launch command.

**Comment by meleu** [comment]
The IP is correct. The problem was only the port... I started RetroPie-Manager listening on port 8080. I deleted the `--port` from the `rpmanager.sh` and everything is fine now. ;-)

**Comment by botolo78** [comment]
I think to make it work as you want, you should make your script edit the settings_production.py and change the port as stated in the --port option

**Comment by meleu** [comment]
Yeah! But there are two places to define the listening port. The settings_production.py file (to define the link) and the comand line to actually define the listening port. But let's use the port 8000 and worry about it later...

**Comment by botolo78** [comment]
Ok
